src/wifes_calib.py

```diff
@@ -793,36 +793,55 @@ def derive_wifes_telluric(cube_fn_list,
             ex_data = numpy.loadtxt(extract_in_list[i])
             obs_wave = ex_data[:,0]
             obs_flux = ex_data[:,1]
+        
+        # Filtering nan values in case of missing flux values
+
+        filter_nan = ~numpy.isnan(obs_flux)
+        obs_wave_nan = obs_wave[filter_nan]  
```

Comment:
I'd reverse these names. `obs_wave_nan` sounds like it has nans in it.

I'd be happy with either `obs_wave = obs_wave_nan[filter_nan]`  
or if that's unclear, maybe `filtered_obs_wave = obs_wave[filter_nan]`

Reply:
Yes, that's better. Done!